pkg/image/api/validation/validation.go

```diff
@@ -24,17 +56,31 @@ func ValidateImage(image *api.Image) errors.ValidationErrorList {
 func ValidateImageRepositoryMapping(mapping *api.ImageRepositoryMapping) errors.ValidationErrorList {
 	result := errors.ValidationErrorList{}
 
-	if len(mapping.DockerImageRepository) == 0 {
-		result = append(result, errors.NewFieldRequired("DockerImageRepository", mapping.DockerImageRepository))
+	hasRepository := len(mapping.DockerImageRepository) != 0
+	hasName := len(mapping.Name) != 0
+	switch {
+	case hasRepository:
+		_, _, _, _, err := api.SplitDockerPullSpec(mapping.DockerImageRepository)
+		if err != nil {
+			result = append(result, errors.NewFieldInvalid("dockerImageRepository", mapping.DockerImageRepository, err.Error()))
+		}
+	case hasName:
```

Comment:
If the mapping has a DockerImageRepository but no Name, you won't get a validation error. Was that intentional?